docs/source/guide/graph-feature.rst

```diff
@@ -27,7 +27,7 @@ features (named ``'x'`` and ``'y'`` in line 5 and 8) and one edge feature (named
           ndata_schemes={'x' : Scheme(shape=(3,), dtype=torch.float32)}
           edata_schemes={'x' : Scheme(shape=(,), dtype=torch.int32)})
     >>> # different names can have different shapes
-    >>> g.ndata['y'] = th.randn(g.num_nodes(), 5)
+    >>> g.ndata['y'] = th.randn(g.num_edges(), 5)
```

Comment:
Is this correct?

Reply:
Some fix the EN version, but my branch did not.
Changed.